crates/api_decl/src/namespaces/zkos.rs

```diff
@@ -0,0 +1,13 @@
+use jsonrpsee::core::RpcResult;
+use jsonrpsee::proc_macros::rpc;
+
+/// API bindings for the `zkos` experimental namespace.
+#[rpc(server, namespace = "zkos")]
+pub trait ZKOSNamespace {
```

Comment:
Nit: acronyms are usually treated as one word in camel case, so `ZkOsNamespace` instead of `ZKOSNamespace`